presto-hive/pom.xml

```diff
@@ -328,6 +328,20 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.10</version>
```

Comment:
add versioned dependency to main `pom.xml` (including exclusions). Therefore you won't need to specify version here

Reply:
Done

Reply:
Easier said than done... This library (and some of it's dependencies, such as httpcore) is pulled in different projects in Presto in different versions, usually as a dependency for some other library. I ended up in situation when code compiles, but tests fails. I'll leave explicit version here...